app/assets/stylesheets/base/_surveyjs.scss

```diff
@@ -190,4 +247,111 @@ $radio-checkbox-width-height: rem-calc(39);
       fill: $grey-md-dark;
     }
   }
+
+  .sv-file {
+    @include input-standard-width;
+    background-color: $white;
+    border: rem-calc(1) solid $grey-md-dark;
+    padding: rem-calc(7) rem-calc(7) rem-calc(7) rem-calc(22);
+
+    display: inline-flex;
+    align-items: center;
+    justify-content: space-between;   
+    
+    // file upload button and placeholder text outer div
+    &__decorator {
+      padding: 0;
+
+      display: flex;
+      order: 2;
+    }
+
+    // file upload button and placeholder text inner div
+    &__wrapper {
+      background-color: $white;
+      margin: 0;
+
+      display: flex;
+      align-items: center;
+
+      transform: none;
+    }
+
+    // file upload button
+    &__choose-btn {
+      @include button-accent-condensed-taller;
+      border-radius: 0;
+      width: rem-calc(235);
+  
+      justify-content: center;
+      order: 2;
+    }
+
+    // placeholder text
+    &__no-file-chosen {
+      font-size: rem-calc(20);
+      margin-right: rem-calc(20);
+      margin-left: 0;
+      color: $grey;
+
+      order: 1;
+    }
+
+    // uploaded file name with delete button outer div
+    &> div:not(.sv-file__decorator) {
+      margin-right: rem-calc(30);
+
+      order: 1;
+    }
+
+    // uploaded file name with delete button inner span
+    &__preview {
+      margin-top: 0;
+      padding-right: 0;
+
+      display: flex;
+      align-items: center;
+
+      // uploaded file name
+      .sv-file__sign {
+        font-size: rem-calc(20);
+        margin-right: rem-calc(10);
+
+        order: 1;
+      }
+
+      // delete button outer div
+      &> :not(.sv-file__sign) {
+        order: 2;
+      }
+    }
+
+    // delete button icon
+    &__remove-svg {
+      fill: $danger-dark;
+
+      display: flex;
+
+      position: static;
+    }
```

Comment:
This looks great but when a graphics file is uploaded and a pre-view is available the close button gets pushed bottm right :(
<img width="399" alt="image" src="https://user-images.githubusercontent.com/38047481/152698756-788e8035-49f3-434c-b2f4-560d85d4c89f.png">

Reply:
Removed preview from second file upload